sys/kern/kern_mbuf.c

```diff
@@ -717,7 +719,7 @@ mb_ctor_clust(void *mem, int size, void *arg, int how)
 #endif
 	m = (struct mbuf *)arg;
 	if (m != NULL) {
-		m->m_ext.ext_buf = (char *)mem;
+		m->m_ext.ext_buf = cheri_csetbounds((char *)mem, size);
```

Comment:
I wonder if you want a macro to set ext_buf and ext_size along with bounding like I did with iovecs?